src/Explorer.js

```diff
@@ -1532,7 +2147,7 @@ class RootView extends React.PureComponent<RootViewProps, {}> {
 
   render() {
     const {
-      operation,
+      operationKind,
```

Comment:
OperationKind might be confusing because graphql uses `kind` as a discriminator field. Can you name it `operationType` instead, since that matches the type name?